.pre-commit-config.yaml

```diff
@@ -125,6 +125,7 @@ repos:
         name: validate release links
         language: script
         entry: utils/validate_release_links.py
+        require_serial: true
```

Comment:
This was causing problems trying to run with --all-files, since multiple processes would all try to modify the same file, and the replacements weren't atomic.